File: src/_1ProgrammingBasicsWithJavaMay2023/_2ConditionalStatements/_2Exercise/_05GodzillaVsKong.java
package _1ProgrammingBasicsWithJavaMay2023._2ConditionalStatements._2Exercise;

import java.util.Scanner;

public class _05GodzillaVsKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double budget = Double.parseDouble(scanner.nextLine());
        int mutesCounts = Integer.parseInt(scanner.nextLine());
        double mutesClothingPrice = Double.parseDouble(scanner.nextLine());
        double decorPrice = budget * 0.10;
        double totalPriceMuteClothes = mutesCounts * mutesClothingPrice;
        if (mutesCounts > 150) {
            totalPriceMuteClothes = totalPriceMuteClothes * 0.90;
        }
        double finalPrice = decorPrice + totalPriceMuteClothes;
        double diff = Math.abs(finalPrice - budget);
        if (finalPrice <= budget){
            System.out.printf("Action!%nWingard starts filming with %.2f leva left.", diff);
        } else {
            System.out.printf("Not enough money!%nWingard needs %.2f leva more.", diff);
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_2ConditionalStatements/_2Exercise/_05GodzillaVsKong.js
function godzillaVsKong(budged, numberOfActors, priceForClothesForOneActor) {
    const PERCENT_FOR_DECOR_PRICE = 0.10;
    const PERCENT_DISCOUNT_FOR_MORE_THAN_150_ACTORS = 0.10;

    const priceForDecor = budged * PERCENT_FOR_DECOR_PRICE;
    let priceForClothes = priceForClothesForOneActor * numberOfActors;

    if (numberOfActors > 150) {
        priceForClothes -= priceForClothes * PERCENT_DISCOUNT_FOR_MORE_THAN_150_ACTORS;
    }

    const totalPrice = priceForClothes + priceForDecor;
    const diff = Math.abs(budged - totalPrice);

    if (totalPrice <= budged) {
        console.log(`Action!`);
        console.log(`Wingard starts filming with ${diff.toFixed(2)} leva left.`);
    } else {
        console.log(`Not enough money!`);
        console.log(`Wingard needs ${diff.toFixed(2)} leva more.`);
    }
}